components/learn/TheCarefulExplanationForAdvanced.vue

```diff
@@ -1,11 +1,13 @@
 <template>
```

Comment:
I thought about it, but it is harder to make someone to update a custom JSON rather than HTML. However, perhaps we can leverage content/nuxt for doing this during importation. Let's see later.